cmd/image/qcow2ova/prep/prepare.go

```diff
@@ -83,6 +83,39 @@ func prepare(mnt, volume, dist, rhnuser, rhnpasswd, rootpasswd string) error {
 		return fmt.Errorf("unable to handle the %s filesystem for %s", fsType, partDev)
 	}
 
+	fstabPath := filepath.Join(mnt, "etc", "fstab")
+
+	//get the boot partition name
+	deviceuuid, err := bootDeviceuuid(fstabPath)
+	if err != nil {
+		return err
+	}
+
+	if deviceuuid != "" {
+		uuid := strings.Split(deviceuuid, "=")[1]
```

Comment:
instead of this, can we populate all this info in that `bootDeviceuuid` function itself in a structure?